sentences/de/cover_HassOpenCover.yaml

```
@@ -2,4 +2,6 @@ language: de
 intents:
   HassOpenCover:
     data:
-      - sentences: []
+      - sentences:
+          - "öffne <area> <cover_variants>"
+          - "öffne <area> <cover_variants>"
```

This is a duplicate and should probably be:
```suggestion
          - "öffne <area> <cover_variants>"
          - "öffne <cover_variants> <area>"
```